app/helpers/application_helper.rb

```diff
@@ -15,4 +15,8 @@ def levy_as_string(levy_in_pence)
   def task_status(task)
     task.status == 'completed' ? 'Complete' : "Due by #{task.due_on}"
   end
+
+  def task_period_date(task)
+    Date.new(task.period_year, task.period_month).to_s(:month_year)
```

Comment:
Can you reword the commit message? It says "We now show it a 8 August" whereas it'll actually be "August 2018".